nsxt/resource_nsxt_policy_bgp_neighbor.go

```diff
@@ -458,6 +500,19 @@ func resourceNsxtPolicyBgpNeighborRead(d *schema.ResourceData, m interface{}) er
 	}
 	d.Set("route_filtering", rFilters)
 
+	var neighborLocalAsConfigs []interface{}
+	if obj.NeighborLocalAsConfig != nil {
+		localAsConfig := make(map[string]interface{})
+		if obj.NeighborLocalAsConfig.LocalAsNum != nil {
```

Comment:
Is the check above "just to be sure" or there could be cases where localAsNum is nil?